packages/tupaia-web/src/layout/ProjectCardList/ProjectCard.tsx

```diff
@@ -127,9 +129,18 @@ export const ProjectPendingLink = () => (
     Approval in progress
   </OutlineLink>
 );
-export const ProjectAllowedLink = ({ url }: LinkProps) => (
-  <BaseLink to={url}>View project</BaseLink>
-);
+
+export const ProjectAllowedLink = ({ url }: LinkProps) => {
```

Comment:
Sorry, this was my bad, I should have mentioned, you can just get `ProjectAllowedLink` to accept another param, as well as `url`, to be like `isLandingPage` or something like that and set the target based on that value. Then in the landing pages, where this is used, pass in `isLandingPage` as `true`